modules/packages/Zarr.chpl

```diff
@@ -141,6 +142,10 @@ module Zarr {
       throw new Error("Expected entries of type %s. Found %s".format(dtype:string, chplType));
   }
 
+  private proc validateCompressor(compressor) throws {
+    const supportedCompressors = ["blosclz", "lz4", "lz4hc", "zlib", "zstd"];
+    if supportedCompressors.find(compressor) == -1 then throw new Error("Unsupported compressor: %s".format(compressor));
```

Comment:
I'd lean towards making this an IllegalArgumentError.

It might also be useful to list the supported values here, so that the user doesn't have to hunt down documentation.